includes/blocks/class-sensei-learner-courses-block.php

```diff
@@ -50,8 +50,12 @@ public function render( $attributes, $content ): string {
 			]
 		);
 
-		if ( isset( $attributes['options']['layoutView'] ) && 'grid' === $attributes['options']['layoutView'] ) {
-			$class .= ' wp-block-sensei-lms-learner-courses--is-grid-view';
+		if ( isset( $attributes['options']['layoutView'] ) ) {
```

Comment:
Also, would be cleaner to just concatenate the value in the class name to remove some lines?

Reply:
> I think we introduced `layoutView` in the first version of this block, right?

Yes, it's been there since inception and the [default is `list`](https://github.com/Automattic/sensei/blob/363bc3f5aca7ef880b0b5b621be96c7d6c89f705/assets/blocks/learner-courses-block/block.json#L11).

I simplified this code in https://github.com/Automattic/sensei/pull/4243/commits/54d6028ddc9a06970fd8e9c1c4d58d27bf983586.